legacyman_parser/parse_classes_of_country.py

```diff
@@ -142,20 +142,21 @@ def create_new_class_with_extracted_subcategory(row: PageElement, country: str,
     if has_tonal:
         tonal_href = urljoin(parsed_url, extract_tonal_href(columns[0]))
     # filter for columns that have 7 cells
-    if (len(columns) == 7):
+    if len(columns) == 7:
         CLASS_COLLECTION.append(ClassU(columns[0].text,
-                                    current_subtype,
-                                    country,
-                                    columns[1].text,
-                                    columns[2].text,
-                                    columns[3].text,
-                                    columns[4].text,
-                                    columns[5].text,
-                                    columns[6].text,
-                                    has_tonal,
-                                    tonal_href))
+                                       current_subtype,
+                                       country,
+                                       columns[1].text,
+                                       columns[2].text,
+                                       columns[3].text,
+                                       columns[4].text,
+                                       columns[5].text,
+                                       columns[6].text,
+                                       has_tonal,
+                                       tonal_href))
     else:
-        print('Warning: encounterd class with too few columns', parsed_url)
+        print('\nWarning: encountered class with too few columns. url: {}\n'.format(parsed_url))
```

Comment:
@rnllv - with this way of reporting, the "warning" is displayed inline with the other parsing text, and could get missed.  Could you modify it so that it gets added to a list of discrepancies (or several lists of discrepances) that get reported at the end of the import please?

Reply:
Yep, handled as part of PR #60 (Issue #56). Please approve this PR, and then #60 should be merged without any conflict.